common/config/api.go

```diff
@@ -88,9 +88,12 @@ func APIFlags() []cli.Flag {
 		},
 		cli.DurationFlag{
 			Name:   FLAG_AWS_TIME_BETWEEN_REQUESTS,
-			Value:  10 * time.Millisecond,
+			Value:  DefaultTimeBetweenRequests,
 			EnvVar: ENVVAR_AWS_TIME_BETWEEN_REQUESTS,
-			Usage:  "duration [ns,us (or µs),ms,s,m,h]",
```

Comment:
Without the Usage help, how will a user know how to configure or specify this flag? Additionally, this usage statement follows a pattern we've used [elsewhere](https://github.com/quintilesims/layer0/blob/api-refactor/common/config/cli.go#L24)

Reply:
this isn't client facing